NOnion/Constants.fs

```diff
@@ -83,7 +83,8 @@ module Constants =
     let internal StreamCreationTimeout = TimeSpan.FromSeconds 10.
 
     // Time limit used for http requests
-    let internal HttpResponseTimeout = TimeSpan.FromMinutes 3.
+    let internal HttpGetResponseTimeout = TimeSpan.FromMinutes 3.
```

Comment:
@aarani I think there's 2 typos in this commit msg: you wanted to write POST when you wrote GET, and GET when you wrote POST, right?

Reply:
@aarani nevermind, I think I understand now